test/passphrase/validation.js

```diff
@@ -263,103 +264,211 @@ describe('passphrase validation', () => {
 	});
 
 	describe('validatePassphrase', () => {
+		const passphraseIsValid = [];
+
 		describe('given a valid passphrase', () => {
 			const passphrase =
 				'model actor shallow eight glue upper seat lobster reason label enlist bridge';
 
-			it('should return true', () => {
-				return validatePassphrase(passphrase).should.be.true();
+			it('should return an array without the errors', () => {
```

Comment:
`with no errors`: `without the errors` sounds like there are errors, they just shouldn't be returned.